scripts/lint.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+git grep -l "bin\/bash" | xargs shellcheck -f gcc
```

Comment:
Do we want to lint all files ending in `.sh` instead.

Reply:
The `gcc` formatter makes `:make` inside of `vim(1)` pickup the linting errors.

If we want to use the default formatter, I'll happily change this to use it.